dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateFolderTask.java

```diff
@@ -35,35 +34,33 @@
     private final Ds3Client ds3Client;
     private final String bucketName;
     private final String folderName;
-    private final List<Ds3Object> ds3ObjectList;
     private final LoggingService loggingService;
     private final ResourceBundle resourceBundle;
 
     public CreateFolderTask(final Ds3Client ds3Client,
                             final String bucketName,
                             final String folderName,
-                            final List<Ds3Object> ds3ObjectList,
                             final LoggingService loggingService,
                             final ResourceBundle resourceBundle) {
         this.ds3Client = ds3Client;
         this.bucketName = bucketName;
         this.folderName = folderName;
-        this.ds3ObjectList = ds3ObjectList;
         this.loggingService = loggingService;
         this.resourceBundle = resourceBundle;
     }
 
     @Override
-    protected Object call() throws Exception {
+    protected Void call() throws IOException {
```

Comment:
Why can't this be turned into a runnable?  The executor will take both.

Reply:
Runnable::run() does not seem to allow exceptions.  Leaving as-is until we separate all tasks with respective services in a separate PR